app/javascript/controllers/scrollable_controller.js

```diff
@@ -1,26 +1,25 @@
 import { Controller } from "@hotwired/stimulus"
-import debounce from "utils/debounce"
+import throttle from "utils/throttle"
 
 export default class extends Controller {
   static classes = [ "top", "notTop", "bottom", "notBottom" ]
   static targets = [ "scrollable", "widget" ]
 
-  debouncedCheckScroll = debounce(this.checkScroll, 50)
-
+  throttledCheckScroll = throttle(() => this.checkScroll(), 50)
   checkScroll() {
     const target = this.scrollableTarget
     const isAtTop = target.scrollTop === 0
     const scrollOffset = target.scrollHeight - target.scrollTop
-    const isAtBottom = Math.abs(scrollOffset - target.clientHeight) <= 2 // occasionally these differ by a pixel or two
+    const isAtBottom = Math.abs(scrollOffset - target.clientHeight) <= 40 // occasionally these differ by a few pixels
```

Comment:
I'm certain I needed to change this after fixing the other debounce > throttle, but I think it makes the "sticking scroll to bottom" a bit more reliable